localization/english/traits/elf_evolved_congential_traits_l_english.yml

```diff
@@ -3,23 +3,22 @@
-  # Radiant => Enchanting
-  trait_beauty_good_4:0 "Radiant"
-  trait_beauty_good_male_4:0 "Radiant"
-  trait_beauty_good_female_4:0 "Enchanting"
-  trait_beauty_good_4_desc:0 "An aura of hypnotic grace exudes from them and to stare upon them is to be lost in a dream. The Divine Spark has gifted them with an angelic visage that is beyond compare."
-  trait_beauty_good_4_character_desc:0 "An aura of hypnotic grace exudes from [ROOT.GetCharacter.GetHerHim] and to stare upon [ROOT.GetCharacter.GetHerHim] is to be lost in a dream. The Divine Spark has gifted [ROOT.GetCharacter.GetFirstNameNoTooltip] with an angelic visage that is beyond compare."
+  trait_physique_good_4_desc:0 "Bones and muscles built like iron, yet wielding the careful grace of a nimble dancer; The Divine Spark has intervened to grant them a level of physical perfection once thought only to be achievable in myth."
+  trait_physique_good_4_character_desc:1 "Bones and muscles built like iron, yet wielding the careful grace of a nimble dancer; The Divine Spark has intervened to grant [Root.GetCharacter.GetFirstNameNoTooltip] a level of physical perfection once thought only to be achievable in myth."
+  # Erudite
+  trait_intellect_good_4:0 "Erudite"
+  trait_intellect_good_4_desc:0 "Complex arithmetic is trivial and sophisticated strategic plans are a mere afterthought. The Divine Spark fills the mind with constant inspiration; granting a level of shrewdness that could rival history's greatest scholars."
+  trait_intellect_good_4_character_desc:0 "Complex arithmetic is trivial and sophisticated strategic plans are a mere afterthought. The Divine Spark fills the mind with constant inspiration; granting [ROOT.GetCharacter.GetFirstNameNoTooltip] a level of shrewdness that could rival history's greatest scholars."
+  # Adonis => Aphroditic
+  trait_beauty_good_male_4:0 "Adonic"
+  trait_beauty_good_female_4:0 "Aphroditic"
+  trait_beauty_good_4_desc:0 "Their hyponitic grace exudes a divine aura that demands attention from all who gaze upon them. The Divine Spark has gifted them with an angelic visage that would make gods envious."
+  trait_beauty_good_4_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip]'s hyponitic grace exudes a divine aura that demands attention from all who gaze upon [ROOT.GetCharacter.GetHerHim]. The Divine Spark has gifted [ROOT.GetCharacter.GetHerHim] with an angelic visage that would make gods envious."
 
   # Shrewd lvl 2
-  trait_shrewd_2:0 "Rational"
-  trait_shrewd_2_desc:0 "This character wields the application of logic to achieve their interests."
-  trait_shrewd_2_character_desc:0 "[ROOT.GetCharacter.GetFirstNameNoTooltip] wields the application of logic to achieve their interests."
+  trait_shrewd_2:0 "Sagacious"
+  trait_shrewd_2_desc:0 "A profound wisdom and understanding of the world and its inhabitants has been gifted to them by the Divine Spark."
```

Comment:
Since Fey Agoras is a standard elf tradition and not a Blessing, id prefer we drop the line saying it was gifted to them by the Divine Spark. They achieve this wisdom from attending the debates/discussions in the Agoras